man/man1/pkcsep11_session.1.in

```diff
@@ -55,6 +55,16 @@ displayed or logged out.
 .IP "\fB-h\fP" 10
 show usage information
 
+.SH "ENVIRONMENT VARIABLES"
+.SS "OCK_USER_PIN"
+The token PIN can be specified via the environment variable OCK_USER_PIN.
+If nothing is set, pkcsicsf will prompt for the token PIN interactively.
+.PP
```

Comment:
pkcsicsf -> pkcsep11

Reply:
And also commit msg: pkcssep11_session -> pkcsep11_session

Reply:
ok, will fix it.

Reply:
This paragraph in the man page will be removed completely, because of the env variable rename.
The commit msg will be fixed.